### PR TITLE
Use correct lowerCameCase for JSON field references

### PR DESCRIPTION
**Pull request by tigrannajaryan**
The rule is:

>The keys of JSON objects are field names converted to lowerCamelCase.

The rule was introduced here: https://github.com/open-telemetry/opentelemetry-proto/commit/5a5e750bf984ad6088e8238397003c0d3693ec25

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

**Review by reyang — APPROVED** [review]

File: specification/otlp.md [hunk 1]
```diff
@@ -385,14 +385,14 @@ JSON Protobuf encoded payloads use proto3 standard defined
...
 for mapping between Protobuf and JSON, with the following deviations from that mapping:
 
-- The `trace_id` and `span_id` byte arrays are represented as
+- The `traceId` and `spanId` byte arrays are represented as
   [case-insensitive hex-encoded strings](https://tools.ietf.org/html/rfc4648#section-8);
   they are not base64-encoded as is defined in the standard
   [Protobuf JSON Mapping](https://developers.google.com/protocol-buffers/docs/proto3#json).
-  Hex encoding is used for `trace_id` and `span_id` fields in all OTLP
+  Hex encoding is used for `traceId` and `spanId` fields in all OTLP
   Protobuf messages, e.g., the `Span`, `Link`, `LogRecord`, etc. messages.
```
Comment (bogdandrutu) [inline specification/otlp.md:393]: put exemplars as well to shod metrics examples.

**Review by bogdandrutu — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

File: specification/otlp.md (hunk 1 above)
Comment (tigrannajaryan) [reply specification/otlp.md:393]: Comment on a wrong PR?

Comment (bogdandrutu) [reply specification/otlp.md:393]: `Protobuf messages, e.g., the `Span`, `Link`, `LogRecord`, etc. messages.`

Comment (tigrannajaryan) [reply specification/otlp.md:393]: Do you mean to add to this list the metric `Exemplar` message?

Comment (bogdandrutu) [reply specification/otlp.md:393]: yes

Comment (tigrannajaryan) [reply specification/otlp.md:393]: OK, can do in a separate PR.

**Pull request merged**